star_tides/api/controllers/base_controller.py

```diff
@@ -33,21 +35,24 @@ class Controller(metaclass=ABCMeta):
     and more to come!
     '''
     def execute(self):
+        error_code = None
         try: # pylint: disable=broad-except
             # log_dict = {'class_name': self.__class__.__name__,
             #             'endpoint': request.url,
             #             'request_method': request.method}
-            # TODO: Convert to logging when logs are implemented. Issue 38
+            # TODO: Convert to logging when logs are implemented. Unknown issue
+            # number.
             response = self.process_request()
-
-
-        # TODO: Handle custom exceptions Issue 38
-
-        # TODO: Handle system exceptions Issue 38
-        except Exception as e:
-            raise e
-
-        return response
+        except StarTidesException as e:
+            response = e
+        except Exception as e:  # pylint: disable=broad-except
+            response = {'error': str(e)}
+            error_code = 500
+
+        return build_response(
+            response,
+            status_code=(200 if error_code is None else error_code)
```

Comment:
Do we need this ternary if the default is 200?

Reply:
No, because if None is provided explicitly then None will be the value.